src/FSharpPlus/Control/Collection.fs

```diff
@@ -305,7 +305,7 @@ type GroupBy =
     static member GroupBy (x: list<'T>, f: 'T->'Key, _: list<'Key*list<'T>>, [<Optional>]_impl: GroupBy) = Seq.groupBy f x |> Seq.map (fun (x, y) -> x, Seq.toList  y) |> Seq.toList
     static member GroupBy (x: 'T []   , f: 'T->'Key, _: ('Key*('T [])) []  , [<Optional>]_impl: GroupBy) = Seq.groupBy f x |> Seq.map (fun (x, y) -> x, Seq.toArray y) |> Seq.toArray
 
-    static member inline Invoke    (projection: 'T->'Key) (source: '``C<'T>``) : '``C<'Key * 'C<'T>>`` = 
+    static member inline Invoke    (projection: 'T->'Key) (source: '``Collection<'T>``) : '``Collection<'Key * 'Collection<'T>>`` = 
```

Comment:
Note the ticks in the type signature. The C here is more for documentation rather than an actual type

Reply:
For sure, I’m wondering if @fxdmhtt has some issue with code? Something that does not compile?